odoo/addons/base/models/ir_http.py

```diff
@@ -487,3 +490,39 @@ def _response_by_status(self, status, headers, content):
             return werkzeug.utils.redirect(content, code=301)
         elif status != 200:
             return request.not_found()
+
+    @classmethod
+    def _get_session_store(cls):
+        """
+        Get the session store from the root session store
+        """
+        return http.root.session_store
```

Comment:
> I don't think this method is useful, what would it be used for? If the session_store is replaced by a non-FS-based store, then the entire vacuum method has to be replaced anyway, as it assumes a it's working with a FilesystemSessionStore.

Yes, agreed it might not be that useful. If the goal is to override the session to a non fs one. 

> Why not just use the config's session_dir directly?

Agreed, that's initially what I did. But changed it back based on your review. So I'm a bit confused now. 

https://github.com/odoo/odoo/pull/70063#discussion_r622756318

I always intended with this PR to make it easier to override the filestore and gc without breaking the way it currently works. Anything beyond that would require more changes that may break odoo. But if we're talking about having it in master... then may be it could be a place for more major changes instead of making this slightly better.